frontend/privateOLD/wireguard.conf.tmpl

```diff
@@ -0,0 +1,11 @@
+[Interface]
```

Comment:
how is this used in the code?

Reply:
This file is not used as all files in privateOLD, but wireguard.conf.tmpl is also in the new folder 

The file are used here
https://github.com/aau-network-security/defatt/blob/2a85617eafd083a4f1387bd5689b6d31c5d6d293/frontend/handler.go#L401

Used to make the VPN config for the user.